src/app/public/src/modules/page-anchor/page-anchor.service.ts

```diff
@@ -1,14 +1,67 @@
-import { Injectable} from '@angular/core';
-import { Subject, Observable } from 'rxjs';
+import { Injectable, OnDestroy } from '@angular/core';
 import { StacheNavLink } from '../nav';
+import { BehaviorSubject, Subject } from 'rxjs';
+import { StacheWindowRef } from '../shared';
 
 @Injectable()
-export class StachePageAnchorService {
+export class StachePageAnchorService implements OnDestroy {
+  public anchorStream$ = new Subject <StacheNavLink[]>();
```

Comment:
What is y'all's preference for denoting an observable property? I've seen either the suffix `Stream` _or_ the use of the dollar sign `$`, but not both? Whichever style you choose, it would probably be good to apply the style to the public property `anchor` (e.g., `anchorStream` or `anchor$`) on the `page-anchor.component.ts` class, too.

Reply:
Is there already a standard pattern used by SKY UX at this time?

Reply:
We've decided to follow the SKY UX pattern of `<property>Stream`.